### PR TITLE
added code for jwt https server

### PR DESCRIPTION
**Pull request by aryan16**
**Please provide a description of this PR:**

Implemented jwt https server and updated the kubernetes deployment yaml file.

### CODE REVIEW

File: samples/jwt-server/server.crt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
```
Comment (yangminzhu) [inline samples/jwt-server/server.crt:1]: This seems duplicated?

File: samples/jwt-server/server.key [hunk 2]
```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```
Comment (yangminzhu) [inline samples/jwt-server/server.key:1]: same here

File: samples/jwt-server/testdata/server.crt [hunk 3]
```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
+MIIDRTCCAi2gAwIBAgIUBeTBvkDJcwG0pIYWP/VphzEGWVEwDQYJKoZIhvcNAQEL
+BQAwRjELMAkGA1UEBhMCVVMxCzAJBgNVBAgMAkFaMRMwEQYDVQQKDApBY21lLCBJ
```
Comment (yangminzhu) [inline samples/jwt-server/testdata/server.crt:3]: Could you add a `README.md` in the same directory and document how could others to re-generate the same cert/key?

File: samples/jwt-server/server.crt (hunk 1 above)
Comment (aryan16) [reply samples/jwt-server/server.crt:1]: sorry, I forgot to remove these.

File: samples/jwt-server/testdata/server.crt (hunk 3 above)
Comment (aryan16) [reply samples/jwt-server/testdata/server.crt:3]: sure.

**Review by yangminzhu — APPROVED** [review]
looks good, thanks!

**Comment by aryan16** [comment]
@istio/wg-docs-maintainers could you please help me merge this PR. The PR is already reviewed by a member of Istio Security team @yangminzhu.

**Review by ericvn — APPROVED** [review]

**Pull request merged**